admin/src/scenes/volontaires/components/DocumentPhase1.js

```diff
@@ -0,0 +1,369 @@
+import React, { useState, useEffect } from "react";
+import { Col, Row } from "reactstrap";
+import { toastr } from "react-redux-toastr";
+import api from "../../../services/api";
+import Select from "../components/Select";
+import Documents from "../components/Documents";
+import DndFileInput from "../../../components/dndFileInput";
+import { Formik } from "formik";
+
+import { FILE_STATUS_PHASE1, translateFileStatusPhase1, translate } from "../../../utils";
+import ModalConfirmWithMessage from "../../../components/modals/ModalConfirmWithMessage";
+
+export default function DocumentPhase1(props) {
+  const [young, setYoung] = useState(props.young);
+  const [statusCohesionStayMedical, setStatusCohesionStayMedical] = useState();
+  const [statusAutoTestPCR, setStatusAutoTestPCR] = useState();
+  const [statusImageRight, setStatusImageRight] = useState();
+  const [statusRules, setStatusRules] = useState();
+  const [modal, setModal] = useState({ isOpen: false, onConfirm: null });
+  const [loading, setLoading] = useState(false);
+  const [dataImageRight, setDataImageRight] = useState();
+  const [dataAutoTestPCR, setDataAutoTestPCR] = useState();
+  const [dataRules, setDataRules] = useState();
+  const options = [FILE_STATUS_PHASE1.TO_UPLOAD, FILE_STATUS_PHASE1.WAITING_VERIFICATION, FILE_STATUS_PHASE1.WAITING_CORRECTION, FILE_STATUS_PHASE1.VALIDATED];
+  const cohesionOptions = [
```

Comment:
ce serait pas plutot ca ducoup ?
```suggestion
  const medicalFileOptions = [
```